hepcrawl/pipelines.py

```diff
@@ -27,7 +27,7 @@
 
 from .tohep import item_to_hep
 from .settings import FILES_STORE
-from .utils import RecordFile
+from .utils import CrawlResult, RecordFile
```

Comment:
`utils` wasn't a great place for `ParsedItem`, it's not a great place for `CrawlResult` either.

Reply:
What's your proposal?

Reply:
Since it's part of the API, what about the `api` module?